source/sdk/threading/native/ConditionWin32.ooc

```diff
@@ -0,0 +1,87 @@
+version(windows) {
+
+	import ../Thread
+	import native/win32/[types, errors]
+	import MutexWin32
+	import structs/ArrayList
+
+	include windows
+
+	CreateEvent: extern func (...) -> Handle
+	SetEvent: extern func (Handle) -> Bool
+	CloseHandle: extern func (Handle) -> Bool
+	INFINITE: extern Long
+	WAIT_OBJECT_0: extern Long
+
+	ConditionWin32: class extends WaitCondition {
+		_mutex := Mutex new()
+		_waitingEvents := ArrayList<Handle> new()
+		_waitingForRelease := ArrayList<Handle> new()
+		init: func
+		free: override func {
+			this destroy()
+			this _waitingEvents free()
+			this _waitingForRelease free()
+			this _mutex destroy()
```

Comment:
Since we seem to have given up the idea of compatibility with the original SDK, what do you think about renaming `destroy` to `free` for consistency? It doesn't need doing in this pull request, of course, just asking.